govcd/system.go

```diff
@@ -149,10 +375,8 @@ func QueryPortGroups(vcdCli *VCDClient, filter string) ([]*types.PortGroupRecord
 	return results.Results.PortGroupRecord, nil
 }
 
-// GetExternalNetwork returns ExternalNetwork object if user specifies a valid external network name.
-// If no valid external network is found, it returns an empty
-// ExternalNetwork and no error. Otherwise it returns an error and an empty
-// ExternalNetwork object
+// GetExternalNetwork returns an ExternalNetwork object if user the network name matches an existing one.
```

Comment:
Nit, maybe better: returns *reference* to an ExternalNetwork?

Reply:
fixed